src/pybind/mgr/cephadm/services/nvmeof.py

```diff
@@ -89,6 +89,22 @@ def get_set_cmd_dicts(out: str) -> List[dict]:
                     logger.warning(f'No ServiceSpec found for {dd.service_name()}')
                     continue
 
+                self.mgr.check_mon_command({
+                    'prefix': 'config-key set',
+                    'key': 'mgr/dashboard/nvmeof/mtls_server_cert',
```

Comment:
since cephadm allows multiple NVMEoF services to exist, we'll need to include the service id in the entry at which we're storing this in the config-key store. Otherwise only the cert/key for the most recently configured NVMEoF service will be present.

Reply:
Ah okay. What about storing them using the daemon id?

Reply:
After talking with @oritwas and Aviv today, it looks like the nvmeof doesn't support multiple services (only multiple daemon within the same service). Does it make sense to enforce that (singleton-like service)?

Reply:
updated to store the cert with service name.